---
title: "The Cresko Laboratory at the University of Oregon"
layout: splash
permalink: /about/
author_profile: false
header:
  overlay_color: "#000"
  overlay_filter: "0.25"
  overlay_image: /images/tyee_slough_header.jpg
  caption: "Tyee Slough, Mary's River"
excerpt: "Our academic mission"
---
I am a geneticist who specializes in quantitative evolutionary genomics. Our laboratory studies the developmental genetic and genomic basis of evolution in natural populations. We use the threespine stickleback and zebrafish as the main animal models in the laboratory. We have produced some of the first work that has helped develop stickleback into a model for dissecting the genetic basis of natural variation. We have developed genomic tools (with the support of NIH) such as sequenced Restriction site Associated DNA (RAD) tags that help geneticists apply Next Generation Sequencing (NGS) technologies to biomedical and evolutionary genetic problems. These techniques allow for the efficient identification of thousands of single nucleotide polymorphisms (SNPs) throughout the genomes of models and non-model organisms. We produced the first SNP whole genome-scan for selection in the stickleback genome, and we developed novel Maximum Likelihood (ML) analytical tools for NGS data. Computational biologists and computer scientists in our team have produced software packages for genomic analyses that are used by laboratories around the world for the analysis of big data problems. Our laboratory, with support from NIH, has developed protocols, best practices, and tools for RNA-seq based transcriptomic functional analyses. Our laboratory is a primary contributor to the NIH Center of Excellence in Systems Biology on the Microbial Ecology and Theory of Animals (META). Through this support we have developed the threespine stickleback into a model for the role of natural genetic variation in host-microbe associations, and we have already made key discoveries about the effects of this genetic diversity on immune system and transcriptional responses. Our group presently has a very nice mix of postdoctoral scholars and graduate students, including molecular geneticists with great bench skills, quantitative biologists who are excellent modelers, and computer scientists who are adept at the production of computational pipelines for the analysis of genomic and transcriptomic data.
