---
title: "Research Areas in the Laboratory"
layout: splash
permalink: /research_host_microbe_interactions/
author_profile: false
header:
  overlay_color: "#000"
  overlay_filter: "0.25"
  overlay_image: /images/header2.jpg
  caption: "Photo credit: [**Quick Yeates**]"
excerpt: "Brief descriptions of the major focal areas of research in the laboratory"
---


__Host-microbe interactions__

Surprisingly little is known about the processes by which microbial communities populate, and persist in, host environments. This is an important problem, as recent work makes a strong case for microbial influences on many aspects of human health, such as inflammation of the gastrointestinal tract. We assume that a number of variables, including host genetic variation, differing environmental conditions, and indeterministic factors underlie variation in the microbiota among individual hosts, but understanding the relative importance of these variables and their interactions is best answered through manipulative experiments. To this end, we leverage the standing genetic variation among and within threespine stickleback populations, the ability to make gnotobiotic stickleback, and controlled laboratory studies to ask how host genetic variation influences the microbiota itself, but also host responses to microbes. Specific responses we are measuring include neutrophil influx as a proxy for inflammation in the gut, and complex transcriptional phenotypes using RNA-seq. Future work in the lab will aim to identify variants in the stickleback genome associated with variation in microbial community structure and host responses to microbes. This objective is best accomplished initially through mapping crosses and GWAS, approaches with which members of the lab are very familiar. Upon discovery of candidate causal loci and regulatory networks, we plan to perturb these systems via CRISPR mutagenesis to verify phenotypic consequences. Our work on host-microbe interactions is an integral component of the NIH-funded META Center for Systems Biology here at the University of Oregon.
