---
title: "Research Areas in the Laboratory"
layout: splash
permalink: /research_popgen_genome/
author_profile: false
header:
  overlay_color: "#000"
  overlay_filter: "0.25"
  overlay_image: /images/header2.jpg
  caption: "Photo credit: [**Quick Yeates**]"
excerpt: "Brief descriptions of the major focal areas of research in the laboratory"
---

__Population genomics and genome evolution__

Beneath the phenomena of evolutionary adaptation lies a complex lattice in which population level processes intersect with developmental and genomic constraints.
The Cresko Lab explores these multilevel interactions in the threespine stickleback fish, which has many attractive advantages: countless replicate natural 
experiments in adaption from marine to freshwater, phenotypic radiations within a single species, examples of extremely rapid evolution on a contemporary time 
scale with extant ancestral populations, and genetic and developmental tractability in the lab. By richly genotyping wild populations and by generating controlled 
laboratory crosses, we investigate genomic signatures of natural selection, phylogeography, genome evolution and structural variation, genomic architecture, gene 
by environment interaction, and the genic underpinnings of adaptation. Rad-seq, a de novo genome subsampling genotyping method pioneered at UO in a Cresko and 
Johnson lab collaboration, made possible the use of population genetic statistics in continuous scans across the stickleback genome. 
This uncovers the parallel nature of genome evolution underlying parallel phenotypic evolution in stickleback populations, and reveals that a large proportion of 
the stickleback genome bears the signature of natural selection in replicate adaptation to freshwater from marine ancestry. 
In addition, building haplotype blocks from paired end RAD-seq data allows us to use coalescent approaches to investigate the age and history of genomic 
regions under differential selection.
