<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.7.30">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Publications – Cresko Laboratory</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./files/images/favicon.ico" rel="icon">
<script src="site_libs/quarto-html/quarto.js" type="module"></script>
<script src="site_libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-d66e402c951608c8dcf8229348e01f8f.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap-f1c7f5182637a0510364daec132bc460.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<style>html{ scroll-behavior: smooth; }</style>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed quarto-light">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a href="./index.html" class="navbar-brand navbar-brand-logo">
    <img src="./files/images/lab-logo.png" alt="Cresko Laboratory" class="navbar-logo">
    </a>
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">Cresko Laboratory</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./about.html"> 
<span class="menu-text">About</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./projects.html"> 
<span class="menu-text">Projects</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./people.html"> 
<span class="menu-text">Team</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./publications.html"> 
<span class="menu-text">Publications</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./talks.html"> 
<span class="menu-text">Talks</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./posts.html"> 
<span class="menu-text">Blog</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./teaching.html"> 
<span class="menu-text">Teaching</span></a>
  </li>  
</ul>
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/wcresko"> <i class="bi bi-github" role="img" aria-label="GitHub">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://twitter.com/creskolab"> <i class="bi bi-twitter" role="img" aria-label="Twitter">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="mailto:wcresko@uoregon.edu"> <i class="bi bi-envelope" role="img" aria-label="Email">
</i> 
<span class="menu-text"></span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-full page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    
<!-- main -->
<main class="content column-page" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Publications</h1>
</div>



<div class="quarto-title-meta column-page">

    
  
    
  </div>
  


</header>


<div class="publications-container">
<section id="section" class="level2">
<h2 data-anchor-id="section">2023</h2>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa bacteria &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1128/mbio.00219-23">Host genomic variation shapes gut microbiome diversity in threespine stickleback fish</a></p>
</div>
<div class="publication-right">
<p><strong>Small CM</strong>, <strong>Beck EA</strong>, <strong>Currey MC</strong>, Tavalire HF, <strong>Bassham S</strong>, <strong>Cresko WA</strong>. <em>mBio</em> e0021923.</p>
</div>
</div>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa fish &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1111/jfb.15349">Genome-wide analysis facilitates estimation of male contribution in meiotic gynogenesis</a></p>
</div>
<div class="publication-right">
<p><strong>Currey MC</strong>, Walker C, <strong>Bassham S</strong>, <strong>Healey HM</strong>, <strong>Beck EA</strong>, <strong>Cresko WA</strong>. <em>Journal of Fish Biology</em> 102(4):844-855.</p>
</div>
</div>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa database &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.46471/gigabyte.76">Improvements to the Gulf pipefish genome</a></p>
</div>
<div class="publication-right">
<p>Ramesh B, <strong>Small CM</strong>, <strong>Healey H</strong>, Johnson B, Barker E, <strong>Currey M</strong>, <strong>Bassham S</strong>, <strong>Cresko WA</strong>, Jones AG. <em>GigaByte</em> 2023:gigabyte76.</p>
</div>
</div>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa shield-virus &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1007/s11121-023-01495-4">Reaching Latinx Communities with Algorithmic Optimization for SARS-CoV-2 Testing Locations</a></p>
</div>
<div class="publication-right">
<p>Searcy JA, Cioffi CC, Tavalire HF, Budd EL, <strong>Cresko WA</strong>, DeGarmo DS, Leve LD. <em>Prevention Science</em> 24(6):1249-1260.</p>
</div>
</div>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa dna &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1093/g3journal/jkac307">Ovarian transcriptional response to Wolbachia infection in D. melanogaster</a></p>
</div>
<div class="publication-right">
<p>Frantz SI, <strong>Small CM</strong>, <strong>Cresko WA</strong>, Singh ND. <em>G3</em> 13(2):jkac307.</p>
</div>
</div>
</section>
<section id="section-1" class="level2">
<h2 data-anchor-id="section-1">2022</h2>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa dna &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.3389/fgene.2022.925786">Extreme intraspecific divergence in mitochondrial haplotypes makes threespine stickleback an emerging evolutionary model</a></p>
</div>
<div class="publication-right">
<p><strong>Beck EA</strong>, <strong>Bassham S</strong>, <strong>Cresko WA</strong>. <em>Frontiers in Genetics</em> 13:925786.</p>
</div>
</div>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa flask &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1073/pnas.2119602119">Leafy and weedy seadragon genomes connect genic and repetitive DNA to syngnathid biology</a></p>
</div>
<div class="publication-right">
<p><strong>Small CM</strong>, <strong>Healey HM</strong>, <strong>Currey MC</strong>, <strong>Beck EA</strong>, Catchen J, Lin ASP, <strong>Cresko WA</strong>, <strong>Bassham S</strong>. <em>PNAS</em> 119(26):e2119602119.</p>
</div>
</div>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa microscope &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1093/genetics/iyac017">Single-cell Iso-Sequencing enables rapid genome annotation for scRNAseq analysis</a></p>
</div>
<div class="publication-right">
<p><strong>Healey HM</strong>, <strong>Bassham S</strong>, <strong>Cresko WA</strong>. <em>Genetics</em> 220(3):iyac017.</p>
</div>
</div>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa virus-covid &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.2105/AJPH.2022.307037">Effects of a Health Education Intervention for COVID-19 Prevention in Latinx Communities</a></p>
</div>
<div class="publication-right">
<p>Anda S, Budd EL, Halvorson S, Mauricio AM, McWhirter EH, Cioffi CC, Garcia JIR, <strong>Cresko WA</strong>, Leve LD, DeGarmo DS. <em>American Journal of Public Health</em> 112(S9):S923-S927.</p>
</div>
</div>
</section>
<section id="section-2" class="level2">
<h2 data-anchor-id="section-2">2021</h2>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa bacteria &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1128/mBio.00548-21">Shared environment and genetics shape the gut microbiome after infant adoption</a></p>
</div>
<div class="publication-right">
<p>Tavalire HF, Christie DM, Leve LD, Ting N, <strong>Cresko WA</strong>, Bohannan BJM. <em>mBio</em> 12(2):e00548-21.</p>
</div>
</div>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa seedling &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1111/mec.15787">Genome-wide SNP markers reveal population structure and dispersal direction of algal blooms</a></p>
</div>
<div class="publication-right">
<p>Rengefors K, Gollnisch R, Sassenhagen I, Härnström Aloisi K, Svensson M, Lebret K, Čertnerová D, <strong>Cresko WA</strong>, <strong>Bassham S</strong>, Agrawal A. <em>Molecular Ecology</em> 30(4):912-925.</p>
</div>
</div>
</section>
<section id="section-3" class="level2">
<h2 data-anchor-id="section-3">2019</h2>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa bacteria &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1128/mSystems.00331-19">Highly reproducible 16S sequencing facilitates measurement of host genetic influences on the stickleback gut microbiome</a></p>
</div>
<div class="publication-right">
<p><strong>Small CM</strong>, <strong>Currey M</strong>, <strong>Beck EA</strong>, <strong>Bassham S</strong>, <strong>Cresko WA</strong>. <em>mSystems</em> 4(4):e00331-19.</p>
</div>
</div>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa fish &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1093/biolinnean/blz106">Genetic divergence outpaces phenotypic evolution among threespine stickleback populations</a></p>
</div>
<div class="publication-right">
<p><strong>Currey MC</strong>, <strong>Bassham SL</strong>, <strong>Cresko WA</strong>. <em>Biological Journal of the Linnean Society</em> 128(2):415-434.</p>
</div>
</div>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa fish &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1111/ede.12273">Developmental timing differences underlie armor loss across threespine stickleback populations</a></p>
</div>
<div class="publication-right">
<p>Petersen AM, <strong>Small CM</strong>, Yan YL, Wilson C, Batzel P, Bremiller RA, Buck CL, von Hippel FA, <strong>Cresko WA</strong>, Postlethwait JH. <em>Evolution and Development</em> 19(6):231-243.</p>
</div>
</div>
</section>
<section id="section-4" class="level2">
<h2 data-anchor-id="section-4">2017</h2>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa dna &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1093/gbe/evx014">Host genotype and microbiota contribute asymmetrically to transcriptional variation</a></p>
</div>
<div class="publication-right">
<p><strong>Small CM</strong>, Milligan-Myhre K, <strong>Bassham S</strong>, Guillemin K, <strong>Cresko WA</strong>. <em>Genome Biology and Evolution</em> 9(3):504-520.</p>
</div>
</div>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa vial &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1016/j.ygcen.2016.10.014">Exogenous iodide ameliorates perchlorate-induced thyroid phenotypes in stickleback</a></p>
</div>
<div class="publication-right">
<p>Gardell AM, von Hippel FA, Adams EM, Dillon DM, Petersen AM, Postlethwait JH, <strong>Cresko WA</strong>, Buck CL. <em>General and Comparative Endocrinology</em> 243:60-69.</p>
</div>
</div>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa fish &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1093/biolinnean/blx111">Ancient stickleback mtDNA lineages are not associated with phenotypic or nuclear genetic variation</a></p>
</div>
<div class="publication-right">
<p>Lescak EA, Wund MA, <strong>Bassham S</strong>, Catchen J, Prince D, Lucas R, Dominguez G, von Hippel FA, <strong>Cresko WA</strong>. <em>Biological Journal of the Linnean Society</em> 122(3):579-588.</p>
</div>
</div>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa users &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1007/s11121-017-0816-y">Naturalistic Experimental Designs as Tools for Understanding Genes and Environment</a></p>
</div>
<div class="publication-right">
<p>Leve LD, Neiderhiser JM, Harold GT, Natsuaki MN, Bohannan BJM, <strong>Cresko WA</strong>. <em>Prevention Science</em> 1-11.</p>
</div>
</div>
</section>
<section id="section-5" class="level2">
<h2 data-anchor-id="section-5">2016</h2>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa dna &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1186/s13059-016-1126-6">The genome of the Gulf pipefish enables understanding of evolutionary innovations</a></p>
</div>
<div class="publication-right">
<p><strong>Small CM</strong>, <strong>Bassham S</strong>, Catchen J, Amores A, Fuiten AM, Brown RS, Jones AG, <strong>Cresko WA</strong>. <em>Genome Biology</em> 17:258.</p>
</div>
</div>
</section>
<section id="section-6" class="level2">
<h2 data-anchor-id="section-6">2015</h2>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa flask &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1073/pnas.1512020112">Evolution of stickleback in 50 years on earthquake-uplifted islands</a></p>
</div>
<div class="publication-right">
<p><strong>Bassham S</strong>, Catchen J, Lescak E, von Hippel FA, <strong>Cresko WA</strong>. <em>PNAS</em> 112(52):E7204-E7212.</p>
</div>
</div>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa vial &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1016/j.ygcen.2014.11.015">Perchlorate disrupts embryonic androgen synthesis without changing thyroid hormone</a></p>
</div>
<div class="publication-right">
<p>Petersen AM, Dillon D, Bernhardt RR, Torunsky R, Postlethwait JH, von Hippel FA, Buck CL, <strong>Cresko WA</strong>. <em>General and Comparative Endocrinology</em> 210:130-144.</p>
</div>
</div>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa fish &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1093/jhered/esv015">Effects of Crossovers Between Homeologs on Inheritance in Polyploid-Derived Salmonid Fishes</a></p>
</div>
<div class="publication-right">
<p>Allendorf FW, <strong>Bassham S</strong>, <strong>Cresko WA</strong>, Limborg MT, Seeb LW, Seeb JE. <em>Journal of Heredity</em> 106(3):217-227.</p>
</div>
</div>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa vial &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1016/j.ygcen.2015.02.022">Developmental timing of sodium perchlorate exposure alters angiogenesis and maturation</a></p>
</div>
<div class="publication-right">
<p>Furin CG, von Hippel FA, Postlethwait JH, Buck CL, <strong>Cresko WA</strong>, O’Hara TM. <em>General and Comparative Endocrinology</em> 219:24-35.</p>
</div>
</div>
</section>
<section id="section-7" class="level2">
<h2 data-anchor-id="section-7">2013</h2>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa map-location-dot &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1111/mec.12330">The population structure and recent colonization history of Oregon threespine stickleback</a></p>
</div>
<div class="publication-right">
<p>Catchen J, <strong>Bassham S</strong>, Wilson T, <strong>Currey M</strong>, O’Brien C, Yeates Q, <strong>Cresko WA</strong>. <em>Molecular Ecology</em> 22(11):2864-2883.</p>
</div>
</div>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa code &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1111/mec.12354">Stacks: an analysis tool set for population genomics</a></p>
</div>
<div class="publication-right">
<p>Catchen J, Hohenlohe PA, <strong>Bassham S</strong>, Amores A, <strong>Cresko WA</strong>. <em>Molecular Ecology</em> 22(11):3124-3140.</p>
</div>
</div>
</section>
<section id="section-8" class="level2">
<h2 data-anchor-id="section-8">2012</h2>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa fish &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1111/j.1525-142X.2012.00550.x">Developmental dissociation in morphological evolution of the stickleback opercle</a></p>
</div>
<div class="publication-right">
<p>Kimmel CB, Hohenlohe PA, Ullmann B, <strong>Currey M</strong>, <strong>Cresko WA</strong>. <em>Evolution and Development</em> 14(4):326-337.</p>
</div>
</div>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa dna &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1098/rstb.2011.0245">Extensive linkage disequilibrium and parallel adaptive divergence across stickleback genomes</a></p>
</div>
<div class="publication-right">
<p>Hohenlohe PA, <strong>Bassham S</strong>, <strong>Currey M</strong>, <strong>Cresko WA</strong>. <em>Philosophical Transactions of the Royal Society B</em> 367(1587):395-408.</p>
</div>
</div>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa book &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1007/978-1-61779-870-2_14">Population genomic analysis of model and nonmodel organisms using sequenced RAD tags</a></p>
</div>
<div class="publication-right">
<p>Hohenlohe PA, Catchen J, <strong>Cresko WA</strong>. <em>Methods in Molecular Biology</em> 888:235-260.</p>
</div>
</div>
</section>
<section id="section-9" class="level2">
<h2 data-anchor-id="section-9">2011</h2>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa fish &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1111/j.1558-5646.2010.01195.x">Cryptic genetic variation and body size evolution in threespine stickleback</a></p>
</div>
<div class="publication-right">
<p>McGuigan K, Nishimura N, <strong>Currey M</strong>, Hurwit D, <strong>Cresko WA</strong>. <em>Evolution</em> 65:1203-1211.</p>
</div>
</div>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa dna &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1371/journal.pone.0018561">Local De Novo Assembly of RAD Paired-End Contigs Using Short Sequencing Reads</a></p>
</div>
<div class="publication-right">
<p>Etter PD, Preston JL, <strong>Bassham S</strong>, <strong>Cresko WA</strong>, Johnson EA. <em>PLoS ONE</em> 6(4):e18561.</p>
</div>
</div>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa book &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1007/978-1-61779-228-1_9">SNP discovery and genotyping for evolutionary genetics using RAD sequencing</a></p>
</div>
<div class="publication-right">
<p>Etter PD, <strong>Bassham S</strong>, Hohenlohe PA, Johnson EA, <strong>Cresko WA</strong>. <em>Methods in Molecular Biology</em> 772:157-178.</p>
</div>
</div>
</section>
<section id="section-10" class="level2">
<h2 data-anchor-id="section-10">2010</h2>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa dna &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1371/journal.pgen.1000862">Population genomics of parallel adaptation in threespine stickleback using sequenced RAD tags</a></p>
</div>
<div class="publication-right">
<p>Hohenlohe PA, <strong>Bassham S</strong>, Etter PD, Stiffler N, Johnson EA, <strong>Cresko WA</strong>. <em>PLoS Genetics</em> 6(2):e1000862.</p>
</div>
</div>
</section>
<section id="section-11" class="level2">
<h2 data-anchor-id="section-11">2008</h2>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa dna &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1371/journal.pone.0003376">Rapid SNP discovery and genetic mapping using sequenced RAD markers</a></p>
</div>
<div class="publication-right">
<p>Baird NA, Etter PD, Atwood TS, <strong>Currey MC</strong>, Shiver AL, Lewis ZA, Selker EU, <strong>Cresko WA</strong>, Johnson EA. <em>PLoS One</em> 3(10):e3376.</p>
</div>
</div>
</section>
<section id="section-12" class="level2">
<h2 data-anchor-id="section-12">2004</h2>
<div class="publication-item">
<div class="publication-icon">
{{&lt; fa flask &gt;}}
</div>
<div class="publication-left">
<p><a href="https://doi.org/10.1073/pnas.0308479101">Parallel genetic basis for repeated evolution of armor loss in Alaskan threespine stickleback</a></p>
</div>
<div class="publication-right">
<p><strong>Cresko WA</strong>, Amores A, Wilson C, Murphy J, <strong>Currey M</strong>, Phillips P, Bell MA, Kimmel CB, Postlethwait JH. <em>PNAS</em> 101(16):6050-6055.</p>
</div>
</div>
</section>
</div>
<hr>
<section id="metrics" class="level2 metrics-section">
<h2 data-anchor-id="metrics">Metrics</h2>
<p><strong>100+</strong> publications | <strong>15,000+</strong> citations | <strong>h-index: 52</strong> | <a href="https://scholar.google.com/citations?user=kMq60q8AAAAJ">{{&lt; ai google-scholar &gt;}} Google Scholar</a></p>
</section>
<p><em>Lab members in bold</em></p>



</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
        const codeEl = trigger.previousElementSibling.cloneNode(true);
        for (const childEl of codeEl.children) {
          if (isCodeAnnotation(childEl)) {
            childEl.remove();
          }
        }
        return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp("https:\/\/wcresko\.github\.io\/lab_website\/");
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
            // target, if specified
            link.setAttribute("target", "_blank");
            if (link.getAttribute("rel") === null) {
              link.setAttribute("rel", "noopener");
            }
            // default icon
            link.classList.add("external");
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
<p>© 2024 Cresko Laboratory</p>
</div>   
    <div class="nav-footer-center">
<p>Built with <a href="https://quarto.org/">Quarto</a></p>
</div>
    <div class="nav-footer-right">
<p><a href="https://uoregon.edu">University of Oregon</a></p>
</div>
  </div>
</footer>




<script src="site_libs/quarto-html/zenscroll-min.js"></script>
</body></html>